src/main/java/org/folio/circulation/domain/ConfigurationService.java

```diff
@@ -1,32 +1,33 @@
 package org.folio.circulation.domain;
 
-import static org.folio.circulation.domain.MultipleRecords.from;
-
-import java.lang.invoke.MethodHandles;
-import java.util.Collection;
-
+import io.vertx.core.json.JsonObject;
 import org.apache.commons.lang3.StringUtils;
 import org.joda.time.DateTimeZone;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import io.vertx.core.json.JsonObject;
+
+import java.lang.invoke.MethodHandles;
+import java.util.Collection;
+
+import static org.folio.circulation.domain.MultipleRecords.from;
```

Comment:
Typically we prefer static imports above regular imports and the order of imports is still somewhat debatable at this point. I am sure some other editor will autochange it to some other order down the road. I don't think there is a need to change it back (the reorder is relatively small), but seeing this time and time again suggests we need to address this soon (unrelated to this PR). We really could use Checkstyle enforcement here. 🤔

Reply:
@mreno-EBSCO I agree it would be nice to use CheckStyle to check this. Given how complex it might be to configure custom check style rules, should we raise the profile of moving to the Google code style, and then trial adopting CheckStyle?

Reply:
Fix